server/app.js

```diff
@@ -8,40 +9,68 @@ const cors = require('cors');
 const app = express();
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({extended: true}));
-app.use(express.static(__dirname + '/../client/dist'))
+app.use(express.static(path.join(__dirname, '..', 'client', 'dist')))
+app.use('/listing', express.static(path.join(__dirname, '..', 'client', 'dist')));
 app.use(cors());
 
-app.get('/api/carousel', (req, res) => {
-  Product.find({}, (err, products) => {
+// Need to add in rest of the CRUD verbs
+app.get('/listing/*', (req, res) => { res.sendFile(path.join(__dirname, '..', 'client', 'dist', 'index.html')); });
```

Comment:
This is good, but I would recommend splitting this into multiple lines to make it more clear to glance value